contracts/protocore/OriginProtocore.sol

```diff
@@ -109,6 +109,8 @@ contract OriginProtocore is MasterCopyNonUpgradable, GenesisOriginProtocore, Pro
      *
      * @dev Satisfies \pre and \post conditions of
      *      Protocore::proposeLinkInternal().
+	 *
+ 	 * \post Emits LinkProposed event.
```

Comment:
Indentation

Reply:
Fixed. It is looking good on vs code, however wrong here because it was tab. We should make sure not to introduce tabs and always have spaces. Fixed!